src/components/Timer/Timer.tsx

```diff
@@ -57,6 +61,7 @@ export const Timer = (props: TimerProps) => {
     if (!previousPlayTimesUpState && playTimesUp) {
       timesUpSoundObject.on("end", () => setPlayTimesUp(false));
       playTimesUpSound();
+      hideTimerAfterSeconds(15);
```

Comment:
You could extract this number into a constant on top so that we don't have random numbers flying around in the code, but since it is only used once anyways I don't really care that much 👀